src/types/Filter.ts

```diff
@@ -0,0 +1 @@
+export type Filter = 'all' | 'completed' | 'active';
```

Comment:
Consider using an enum instead of a type for better type safety and readability. 🤓

Reply:
done